src/main/java/com/github/zedd7/zhorse/utils/MessageConfig.java

```diff
@@ -150,7 +150,10 @@ private <T> List<String> getFlagContentList(List<T> tokenList) {
 				flagContent = numberFormatter.format(token);
 			}
 			else {
-				flagContent = token.toString();

```

Comment:
consider using Objects.toString(o, nullDefault) instead
https://docs.oracle.com/javase/7/docs/api/java/util/Objects.html#toString%28java.lang.Object,%20java.lang.String%29

Reply:
Thanks for reviewing! Good catch.